test/state/state.cpp

```diff
@@ -12,10 +12,15 @@
 #include <evmone/execution_state.hpp>
 #include <algorithm>
 
+using namespace intx;
+
 namespace evmone::state
 {
 namespace
 {
+constexpr auto SECP256K1N = 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141_u256;
```

Comment:
"GitGuardian Security Checks" flag this as a leaked secred, which is obviosly false positive, but also will go away with the next spec change https://github.com/ethereum/eips/pull/8845